cfbs/git.py

```diff
@@ -177,3 +177,25 @@ def git_discard_changes_in_file(file_name):
         raise CFBSGitError(
             "Failed to discard changes in file '%s'" % file_name
         ) from cpe
+
+
+def git_check_tracked_changes(scope=["all"]):
+    should_commit = False
+    try:
+        result = run(["git", "status", "-s", "-u"], check=True, stdout=PIPE)
+        if "all" in scope:
+            if len(result.stdout) > 0:
+                should_commit = True
+        else:
+            changes = list(
+                map(
+                    lambda m: m.strip().split(" ")[1],
+                    list(filter(None, result.stdout.decode("utf-8").split("\n"))),
+                )
+            )
```

Comment:
@mineralsfree something like this should work, and might be considered more readable :)
```suggestion
            lines = result.stdout.decode("utf-8").split("\n")
            changes = [line.strip().split(" ")[1] for line in lines if line]
```